assets/src/util/math.js

```diff
@@ -8,13 +8,13 @@ const getDecimalPlaceOfFloat = (floatNum) => (floatNum.toString().split('.')[1])
 const average = arr => arr.reduce((acc, cur) => acc + cur, 0) / arr.length
 
 const pearsonCorrelation = (prefs, p1, p2) => {
```

Comment:
I don't see the function `pearsonCorrelation` being used. I just left it there, not sure what was the reason they were left off

Reply:
I think it can be removed